restricted_site_access.php

```diff
@@ -1029,14 +1033,14 @@ public static function deactivation( $network_active ) {
 			foreach ( $sites as $site ) {
 				switch_to_blog( $site->blog_id );
 
-				if ( 2 == get_option( 'blog_public' ) ) {
+				if ( 2 === get_option( 'blog_public' ) ) {
```

Comment:
Since WordPress `get_option` _returns scalar values as strings_ (https://developer.wordpress.org/reference/functions/get_option/) these strict equality checks may not work as expected. For each of these strict comparisons to integers, we can cast the return from get_option with (int) or `intval `

Reply:
Right. Made change for this and for other strict checks as well.